packages/mobile/src/app/App.tsx

```diff
@@ -52,32 +57,42 @@ BigNumber.config({
 // }
 
 export class App extends React.Component {
+  state: State = {
+    reactInitTime: undefined,
+    reactLoadTime: undefined,
+  }
+
+  appStartListener: EmitterSubscription = new NativeEventEmitter().addListener(
```

Comment:
Does this work on both Android and iOS? If only on Android can you leave a comment

Reply:
This should work for both Android and iOS but we are only emitting an event from Android native code at the moment. The hope is that we can use this same listener to capture iOS events once we figure out how to emit an event in iOS native code.